installing/installing_other/install-secured-cluster-other.adoc

```diff
@@ -8,7 +8,12 @@ toc::[]
 
 [role="_abstract"]
 
-You can install {product-title-short} on your secured clusters for platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).
+You can install {rh-rhacs-first} on your secured clusters for the following platforms:
+
+* Amazon Elastic Kubernetes Service (Amazon EKS)
+* Google Kubernetes Engine (Google GKE)
+* Microsoft Azure Kubernetes Service (Microsoft AKS)
+* Azure Resource Manager (ARM)
```

Comment:
This is not a k8s platform, when we say Arm support, we mean the [Arm architecture](https://www.arm.com/architecture), similar to Power (ppc64le architecture) and Z (s390x architecture) are also supported.